view/frontend/templates/boltaccount.phtml

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2018 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+/**
+ * Bolt Account button
+ *
+ * @var $block \Bolt\Boltpay\Block\Js
+ */
+if ($block->shouldDisableBoltCheckout()) return;
+if (!$block->isOrderManagementEnabled()) return;
+if ($block->isBlockAlreadyShown('account')) return;
+$accountJsUrl   = $block->getAccountJsUrl();
+$checkoutKey  = $block->getCheckoutKey();
+?>
+<div class="bolt-account-login"></div>
```

Comment:
It works for desktop, but doesn't for mobile. Will discuss with BX team on Monday

Reply:
Any follow-up here from BX team?

Reply:
I am sorry Matt, I didn't mention that your comment from this thread. I've figured out what wrong was with mobile, it was specifically M2 issue. I've fixed it, now it works for mobile as well.